squire_lib/src/players/mod.rs

```diff
@@ -132,3 +132,68 @@ impl Display for PlayerStatus {
         )
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn new_player_test() {
+        let player = Player::new("Morgan");
```

Comment:
This is failing to compile. In Rust, there is a difference between a string literal, what you have, and a `String`. A string literal is a "string slice", called `&str` in code. String slices are, basically, references to a chunk of memory that contain data that's encoded as text. `String`s contain an allocated string slice (it lives on the heap). They also own that slice, so they can grow dynamically and the backing text is deallocated when the `String` is dropped.

There are about a million ways to convert this literal into a `String`, but probably the easiest why is to use the `Into` trait:
``` suggestion
        let player = Player::new("Morgan".into());
```

Other ways you can handle this are:
 - `"literal".to_owned()`
 - `String::from("literal")`
 - `format!("literal")` (not recommended unless you need to do some string formating too)